docs/apm/transactions.asciidoc

```diff
@@ -5,13 +5,23 @@ TIP: A {apm-overview-ref-v}/transactions.html[transaction] describes an event ca
 The APM agents automatically collect performance metrics on HTTP requests, database queries, and much more.
 
 Selecting a <<services,*service*>> brings you to the *transactions* overview.
-The *transaction duration* and *requests per minute* chart display information on all transactions associated with the selected service.
+The *time spent by span type*, *transaction duration* and *requests per minute* chart display information on all transactions associated with the selected service.
 The *Transactions* table, however, provides only a list of _transaction groups_ for the selected service.
 In other words, this view groups all transactions of the same name together, and only displays one transaction for each group.
 
 [role="screenshot"]
 image::apm/images/apm-transactions-overview.png[Example view of transactions table in the APM UI in Kibana]
 
+*Time spent by span type* -- beta[] Certain agents support breakdown graphs in the APM UI.
+This graph is an easy way to visualize how much time each span is taking within your application.
+The time a transaction took to complete is also recorded and displayed on the chart under the "app" label.
+"App" indicates there was something happening within the application, but we're not sure what.
+This could be a sign that the agent does not have auto-instrumentation for whatever was happening during that time.
+
+It's important to note that if you have asynchronous spans, the total time of all spans might exceed 100%.
```

Comment:
It doesn't really exceed 100%. It's just that the sum of all span self times may exceed the transaction duration. 100% does not correspond to the transaction duration but the sum of all span self times.

Reply:
Ahh, thanks for the clarification Felix